easybuild/easyconfigs/r/R/R-4.0.4-fosscuda-2020b.eb

```
@@ -2155,7 +2155,7 @@ exts_list = [
         'checksums': ['7ffb047692c3717c5fee94fd115d23ce971309aa38aa42cd1e8bef67bc57d49e'],
     }),
     ('mvabund', '4.1.9', {
-        'checksums': ['60d09af4b82fe4945ef0b5aeb85aee2741ccc8a7b3cea26eecef4cf914787ea6'],
+        'checksums': ['fa48c1cabe3a5bed35b0f31b02f55d9f15e57b3945cf3f407ddc9d3c9e29bb30'],
```

Thanks for the fix. We should still keep the previous checksum in case some other user already downloaded the tarball of this extension.
```suggestion
        'checksums': [('60d09af4b82fe4945ef0b5aeb85aee2741ccc8a7b3cea26eecef4cf914787ea6',
                       'fa48c1cabe3a5bed35b0f31b02f55d9f15e57b3945cf3f407ddc9d3c9e29bb30')],
```

---

@Ghepardo since this is a trivial change, I pushed the suggestion myself. I hope that you don't mind.